src/index.html

```diff
@@ -11,7 +11,122 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form>
+      <fieldset class="form__fieldset form-fieldset">
+        <legend class="form-fieldset__title">Personal information</legend>
+        <div class="field">
+          <label for="surname">Surname: </label>
```

Comment:
"Every field should have label, which focuses input on label click", your labels after click not focusing inputs.
Please add an id with the same value as you have in label for attribute and then it should work :)
Do that for the rest form elements also.